backend/app/images.js

```diff
@@ -15,9 +15,16 @@ const Response = require('/opt/nodejs/response.js')
 
 exports.get = async (event, context) => {
   let response = new Response(event)
+
+  let size = event.queryStringParameters['size'] : '1'
```

Comment:
You missed a ? in the ternary operator. This caused the 502.

Reply:
I also updated the master branch travis file so it should not be pushing the changes to lambda when not on master.